iohk/Appveyor.hs

```diff
@@ -0,0 +1,79 @@
+{-# OPTIONS_GHC -Weverything -Wno-unsafe -Wno-implicit-prelude -Wno-semigroup #-}
```

Comment:
I feel like an idiot but I have never seen before the `-Weverything` option (always used `-Wall`) and couldn't find it in the [GHC flags reference](https://downloads.haskell.org/~ghc/8.2.1/docs/html/users_guide/flags.html). Is this a recent addition?

What about the other 3? (sorry for the lame question, it's genuine curiosity!)

Reply:
oddly. `-Wall` doesn't enable all warnings but `-Weverything` does enable everything, and the other 3 just turn a few of the them back off

Reply:
https://gist.github.com/cleverca22/98729e1dd349d31d890c61900bc80396
this is what happens if i just allow everything and remove the `no`'s

Reply:
wow, TIL I guess! Now you picked my curiosity, so I will ask: where did you learn about `-Weverything`? Is this only in some secret haskell devops manual? 😁

Reply:
i learned it from @taktoa